<?xml version="1.0" encoding="UTF-8"?>
<launch>

    <node pkg="octomap_server" type="octomap_server_node" name="octomap_server" output="screen">
		<param name="resolution" value="0.05" />

        <param name="filter_ground" type="bool" value="true" />

		<!-- fixed map frame (set to 'map' if SLAM or localization running!) -->
        <param name="frame_id" type="string" value="odom" />
		<!-- data source to integrate (PointCloud2) -->
        <remap from="cloud_in" to="/camera/depth/points" />
	
    </node>


</launch>

